### PR TITLE
Adding PostgreSQL specific types

### PR DESCRIPTION
**Pull request by msmakouz**

### CODE REVIEW

File: src/Driver/Postgres/Schema/PostgresColumn.php [hunk 1]
```diff
@@ -42,80 +43,167 @@ class PostgresColumn extends AbstractColumn
         'attributes',
     ];
 
+    private const INTERVAL_TYPES = [
```
Comment (roxblnfk) [inline src/Driver/Postgres/Schema/PostgresColumn.php]:
```suggestion
    protected const INTERVAL_TYPES = [
```

File: src/Driver/Postgres/Schema/PostgresColumn.php [hunk 2]
```diff
@@ -42,80 +43,167 @@ class PostgresColumn extends AbstractColumn
...
+        'YEAR TO MONTH',
+        'DAY TO HOUR',
+        'DAY TO MINUTE',
+        'DAY TO SECOND',
+        'HOUR TO MINUTE',
+        'HOUR TO SECOND',
+        'MINUTE TO SECOND',
+    ];
+
+    private const INTERVALS_WITH_ALLOWED_PRECISION = [
```
Comment (roxblnfk) [inline src/Driver/Postgres/Schema/PostgresColumn.php]:
```suggestion
    protected const INTERVALS_WITH_ALLOWED_PRECISION = [
```

File: src/Driver/Postgres/Schema/PostgresColumn.php [hunk 3]
```diff
@@ -42,80 +43,167 @@ class PostgresColumn extends AbstractColumn
...
+
+    protected array $aliases = [
+        'int'            => 'integer',
+        'smallint'       => 'smallInteger',
+        'bigint'         => 'bigInteger',
+        'incremental'    => 'primary',
+        'bigIncremental' => 'bigPrimary',
+        'bool'           => 'boolean',
+        'blob'           => 'binary',
+        'bitVarying'     => 'bit varying',
```
Comment (roxblnfk) [inline src/Driver/Postgres/Schema/PostgresColumn.php:98]: Need to add all magic types as `@method` annotations for the class.

**Review by roxblnfk — APPROVED** [review]

File: src/Driver/Postgres/Schema/PostgresColumn.php (hunk 1 above)
Comment (msmakouz) [reply src/Driver/Postgres/Schema/PostgresColumn.php]: Changed visibility to protected.

File: src/Driver/Postgres/Schema/PostgresColumn.php (hunk 2 above)
Comment (msmakouz) [reply src/Driver/Postgres/Schema/PostgresColumn.php]: Changed visibility to protected.

File: src/Driver/Postgres/Schema/PostgresColumn.php (hunk 3 above)
Comment (msmakouz) [reply src/Driver/Postgres/Schema/PostgresColumn.php:98]: Added

**Review by roxblnfk — APPROVED** [review]

**Pull request merged**